fixtures/webview.py

```diff
@@ -85,3 +85,20 @@ def webview_instance(webview_base_url):
         return url.subdomain
     else:
         return "prod"
+
+
+@pytest.fixture
+def redirecting_books_titles():
+    """Returns all the books from the list of redirecting books to rex
+    """
+
+    here = DATA_DIR + "/redirected_books.txt"
```

Comment:
Use `os.path.join(pathA, pathB)` to join file paths. Did you see my other comment about looking at other examples? There is already a fixture that gets you the list of books.

https://github.com/openstax/cnx-automation/blob/master/fixtures/webview.py#L70-L78